### PR TITLE
add logIn and join pages

### PR DESCRIPTION
**Pull request by daria31v**

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -11,15 +11,22 @@ module.exports = {
...
-      blue_light: '#DBEAE7;'
+      blue: '#3C64B1',
+      blue_light: '#DBEAE7',
+      blue_text: '#87A4DC',
+      grey_text: '#9BADCE',
+      // background: linear-gradient(270.8deg, #3C64B1 0%, #BED3FB 100%);
+
     },
     fontWeight: {
+      small: '300',
```
Comment (pavelsuraba) [inline tailwind.config.ts]: font weights are usually named like this:
100 – Thin.
200 – Extra Light (Ultra Light)
300 – Light.
400 – Normal.
500 – Medium.
600 – Semi Bold (Demi Bold)
700 – Bold.
800 – Extra Bold (Ultra Bold)

File: template.txt [hunk 2]
```diff
@@ -1,2 +1,21 @@
 effect for cards
-transition transform hover:-translate-y-3 motion-reduce:transition-none
\ No newline at end of file
+transition transform hover:-translate-y-3 motion-reduce:transition-none
+
+
+
+// Usage of the component
+const App = () => {
```
Comment (pavelsuraba) [inline template.txt]: What's this?

File: src/app/components/MySvg/MySVG.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+export const MySVGImage: React.FC<React.SVGProps<SVGImageElement>> = (props) => {
```
Comment (pavelsuraba) [inline src/app/components/MySvg/MySVG.tsx:4]: Better to use svg this way: https://blog.logrocket.com/import-svgs-next-js-apps/
Create a component directly in svg file.

File: src/app/login/page.tsx [hunk 4]
```diff
@@ -1,11 +1,91 @@
...
+          <div className="flex justify-center items-center">
+            <button
+              type="submit"
+              className="w-[280px] h-[33px] rounded-[50px] bg-gradient-to-r to-[#2e5fba] via-transparent from-[#E2EAF9] my-3 text-white text-medium font-normal hover:font-bold"
+            >
+              Log In
+            </button>
+          </div>
+          <div className="flex justify-between items-center gap-3 w-full">
+            <div className="h-[1px] bg-blue_text w-full"></div>
```
Comment (pavelsuraba) [inline src/app/login/page.tsx]: h-[1px] -> h-px

File: src/app/registration/page.tsx [hunk 5]
```diff
@@ -1,11 +1,84 @@
...
+import Link from 'next/link'
+
 
 export default function Registration() {
   return (
     <Container>
-      <div>
-        <h1 className="text-ml mt-1 mb-1 text-center uppercase">Joing Us</h1>
+      <div  className="flex justify-center items-center my-36">
+      <div  className="bg-gradient-to-r to-[#b0c7f3] via-transparent from-[#E2EAF9] w-[626px] grid justify-center py-6 rounded-md">
```
Comment (pavelsuraba) [inline src/app/registration/page.tsx]: Seems like registration and login are almost identical. Should be 1 component.

File: src/app/registration/page.tsx [hunk 6]
```diff
@@ -51,26 +52,26 @@ export default function Registration() {
...
-            <div className="h-[1px] bg-blue_text w-full"></div>
+            <div className="h-px bg-blue_text w-full"></div>
           </div>
         </form>
         
           <ul className="flex justify-center items-center my-3 gap-6">
             <li className="bg-white w-10 h-10 rounded-md flex justify-center items-center">
               <Link href="/" className="">
-                <MySVGImage xlinkHref="icon/google.svg" x={0} y={0} width={28} height={28} />
+                <GoogleSvgComponent />
```
Comment (daria31v) [inline src/app/registration/page.tsx:64]: this

Comment (daria31v) [reply src/app/registration/page.tsx:64]: 64-74 lines

**Pull request merged**